src/protobuf-net/Meta/RuntimeTypeModel.cs

```diff
@@ -1018,6 +1018,40 @@ public void SetFrameworkOptions(MetaType from)
             /// </summary>
             public int MetaDataVersion { get { return metaDataVersion; } set { metaDataVersion = value; } }
 
+            /// <summary>
+            /// The company name to burn into the generated assembly
+            /// </summary>
+            public string AssemblyCompanyName { get; set; }

```

Comment:
if we are adding *additional* features here (beyond the `AssemblyVersion` in the `main` branch), we'll need to include those upwards into `main` too; are the other features *needed* for your intent, or were they just "while I'm here..."?

Reply:
Yes, they are definitely needed (including in v2), as far as I'm concerned. Makes total sense to have them in v3+ as well, for when upgrading.

Reply:
fine; I can do a forward port when we have this in a mergeable state (I'm assuming once is enough for you ;p)